WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalError.swift

```diff
@@ -61,6 +61,25 @@ final class CardPresentModalError: CardPresentPaymentsModalViewModel {
     func didTapAuxiliaryButton(in viewController: UIViewController?) { }
 }
 
+// CardPresentPaymentsModalViewModelActions
+extension CardPresentModalError {
+    var primaryButtonViewModel: CardPresentPaymentsModalButtonViewModel? {
+        CardPresentPaymentsModalButtonViewModel(
+            title: primaryButtonTitle,
+            actionHandler: primaryAction)
+    }
+
+    var secondaryButtonViewModel: CardPresentPaymentsModalButtonViewModel? {
+        // TODO: dismissCompletion is not enough for dismissal.
+        // Find another way to dismiss the alert here. We may need a specialised alert provider to fix this one.
+        CardPresentPaymentsModalButtonViewModel(
+            title: secondaryButtonTitle,
+            actionHandler: { [weak self] in
+                self?.dismissCompletion()
+            })
```

Comment:
super nit: updated in https://github.com/woocommerce/woocommerce-ios/pull/12946/commits/898910791d2a3c7c3f886ea80fc1dc5dc47cd89d
```suggestion
            actionHandler: dismissCompletion)
```